modules/client/src/lib/utils.ts

```diff
@@ -86,3 +86,19 @@ export const createRandom32ByteHexString = (): string => {
 
 export const createPaymentId = createRandom32ByteHexString;
 export const createPreImage = createRandom32ByteHexString;
+
+export function removeTrailing0x(str: string) {
+  if (str.startsWith("0x")) {
+    return str.substring(2);
+  } else {
+    return str;
+  }
+}
```

Comment:
1. This should be called remove**Leading**0x
2. This prob doesn't need to be a separate function bc we can just do `.replace(/^0x/, '')`

Reply:
Good catch -- completely missed this. I don't mind having an extra function for it, but we should update the name

Reply:
Makes sense, I've renamed and refactored these hex utils in commit https://github.com/ConnextProject/indra/pull/815/commits/2cd997f308261f25d2912b592d19a2b958cd6169